hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/EdgeLabelBuilder.java

```diff
@@ -149,6 +149,71 @@ public EdgeLabel create() {
         });
     }
 
+    private boolean hasSameProperties(EdgeLabel existedEdgeLabel) {
+        HugeGraph graph = this.graph();
+        Id sourceId = graph.vertexLabel(this.sourceLabel).id();
+        if (! existedEdgeLabel.sourceLabel().equals(sourceId)) {
```

Comment:
There should no space between `!` and `existedEdgeLabel.sourceLabel().equals(sourceId)` in our code style